packages/amplify_authenticator/lib/src/widgets/component.dart

```diff
@@ -50,7 +50,7 @@ abstract class StatelessAuthenticatorComponent extends StatelessWidget {
 /// {@template amplify_authenticator.authenticator_component}
 /// Base class for all stateful Authenticator components.
 /// {@endtemplate}
-abstract class AuthenticatorComponent<T extends AuthenticatorComponent<T>>
+abstract class AuthenticatorComponent<T extends StatefulWidget>
```

Comment:
These changes are needed to make the changes to AuthenticatorFormState class. Without the the changes to the AuthenticatorFormState class, `widget` within _SignUpFormState is of type `AuthenticatorForm` instead of the more specific `SignUpForm`.

That is an issue because `useRunTimeFields` doesn't exist on `AuthenticatorForm`. It exists on `SignUpForm`

Reply:
Ohh, I see